octue/twined
# Refactor: Rest of tests

## Summary
Refactor the rest of the tests (follows https://github.com/octue/twined/pull/52):
* Move JSON fixtures into test files as strings
* Preserve the testing of using actual twine files via new method on `BaseTestCase` (note: Python 3.8 operator used - should this be avoided to keep backwards compatibility with <3.8? See https://github.com/octue/twined/pull/54/commits/aafef4cd6c834fccd00d5c316eee18d74b41ade5)
* Remove single-use variables

Fixes:
* Fix missed merge problem with strand attributes

Other:
* Add refactor branch type to allowed precommit types
